packages/react-ui/src/components/molecules/UploadField.js

```diff
@@ -47,23 +48,25 @@ const useStyles = makeStyles((theme) => ({
   }
 }));
 
-function UploadField(props) {
+function UploadField({ buttonText, accept, files, ...props }) {
```

Comment:
For consistency, could be worth destructuring the other ones: `multiple`, `onChange` and `size`

Reply:
I destructured these ones to avoid some console warnings regarding propagation of props:

`React does not recognize the 'buttonText' prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `buttontext` instead. If you accidentally passed it from a parent component, remove it from the DOM element`

`buttonText, accept, files` are custom props, the other ones are from Mui TextField, do you think it's better to destructure them too?